duckencoder.py

```diff
@@ -22,160 +23,171 @@ def readResource(filename):
                         if len(l) == 0:
                                 continue
 
-                        splitted = l.split("=")
-                        key = splitted[0].strip()
-                        val = splitted[1].strip()
-                        result_dict[key] = val
+                        key, _, val = l.partition("=")
+                        result_dict[key.strip()] = val.strip()
 
                 return result_dict
 
         @staticmethod
+        def altReadResource(filename):
+                with open(filename, "r") as f:
+                        lines = (line.strip().split("//")[0].partition("=") for line in f)
+                        return {line[0].strip(): line[-1].strip() for line in lines if line}
+
+        @staticmethod
         def parseScriptLine(line, keyProp, langProp):
                 result = ""
 
                 # split line into command and arguments
-                instr = line.split(" ", 1)
-                instr[0] = instr[0].strip()
-                if len(instr) > 1:
-                        instr[1] = instr[1].strip()
+                cmd, _, args = line.partition(" ")
+                cmd = cmd.strip()
+                args = args.strip()
 
```

Comment:
Same as above, renaming `instr[0]` to `cmd` and `instr[1]` to `args` makes sense.